### ISSUE TITLE
Hash values discrepancy for MacOS and Linux installers

### ISSUE THREAD

**Issue by smu160** [conda/conda#10574]
## Current Behavior

The hash values for Python 3.9 Miniconda3 MacOSX 64-bit bash and Python 3.9 Miniconda3 Linux 64-bit seem to be incorrect.

#### macOS
```Bash
$ wget https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-MacOSX-x86_64.sh
$ shasum -a 256 Miniconda3-py39_4.9.2-MacOSX-x86_64.sh
```
Output:
`b3bf77cbb81ee235ec6858146a2a84d20f8ecdeb614678030c39baacb5acbed1  Miniconda3-py39_4.9.2-MacOSX-x86_64.sh`

#### Linux
```Bash
$ wget https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh
$ shasum -a 256 Miniconda3-py39_4.9.2-Linux-x86_64.sh 
```
Output:
`536817d1b14cb1ada88900f5be51ce0a5e042bae178b5550e62f61e223deae7c  Miniconda3-py39_4.9.2-Linux-x86_64.sh
`

## Expected Behavior

This does not correspond with the value for the Python 3.9 macOS installer: `0658832393a50f5a578bcad825304886` or the value for the Python 3.9 Linux installer: `b4e46fcc8029e2cfa731b788f25b1d36` available on the [webpage](https://docs.conda.io/en/latest/miniconda.html#linux-installers).

Assuming those are supposed to be SHA256 hashes, should they be 64 chars long? The aforementioned ones consist of only 32 chars.

For further context, here is a current screenshot of the webpage:

<img width="1101" alt="Screen Shot 2021-03-26 at 11 58 21" src="https://user-images.githubusercontent.com/8347475/112661146-9423a000-8e2c-11eb-9176-6b270bf0a483.png" />

## Environment Information
N/A

---
I may be doing something wrong here on my end, so please feel free to let me know if that is the case. 

Thank you!

**Comment by compumetrika** [comment]
I just came to post the same thing; I am seeing this as well. For posterity, here is the Linux miniconda hash for 3.9:

    $ wget https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh
    $ sha256sum Miniconda3-py39_4.9.2-Linux-x86_64.sh                                           
    536817d1b14cb1ada88900f5be51ce0a5e042bae178b5550e62f61e223deae7c  Miniconda3-py39_4.9.2-Linux-x86_64.sh

**Comment by maple3142** [comment]
It seems conda switched to md5, but didn't update the column name?
![image](https://user-images.githubusercontent.com/9370547/113395714-dae53d00-93cc-11eb-9408-29768523c5c9.png)

**Comment by chenghlee** [comment]
The documentation webpage incorrectly showed MD5, rather than SHA256, checksums for the installer; this was fixed in conda/conda-docs#723. Closing this issue.
